### ISSUE TITLE
Ultra vs Express on a 1080 monitor

### ISSUE THREAD

**Issue by chodthewacko** [chtaylo2/Roku-GooglePhotos#303]
Here's a weird one.

I have a viewsonic VP2468 which I am using as a digital photo frame hanging on the wall.  (It's a very clean monitor!  No outside buttons/logos/etc).  It is a native 1920x1080 monitor.

I have a Roku Ultra, and a roku express (2019).  I'd like to use the roku express for the picture frame as it can be USB powered and it's cleaner.

Here is the story in full - it's a bit wordy but bear with me.

1)  I originally used photoview on my Roku Ultra  on my 4K tv to view my google pictures as a screensaver.
2)  Currently, I use a chromecast to view pictures on the photo frame.

I'd always known that the chromecast downloads pictures as 720p resolution.  I was still using it however, since it's convenient.
Just out of curiosity, I set the Roku Ultra and photoview to change all settings down to 1080p (download pictures at 1080p, view at 1080p) and plugged it into the other HDMI port.

The pictures on the Roku Ultra looked FAR better, as you would kind of expect.

Today I got a Roku Express (2019) and set everything the same as the Roku Ultra (1080p everything).  To my amazement, the pictures on photoview were pretty bad.  In come cases they weren't even as good as the chromecast!    What a dissapointment.

So my question is this:  Why is there such a difference?
Is it because the Roku Ultra has downloaded/cached pictures at 4K and is interpolating/downsizing them to 1080p?     If so, there should be an option to download the pictures at full resolution and then view at whatever resolution.

**Comment by chtaylo2** [comment]
Thats hard to say. When you select a SD/HD/FHD/etc, this is the size that's requested from Google's API. There is no resolution upgrades or downgrades taking place on the Roku device itself.

Image quality should be impacted by a number of things, most often limited bandwidth -- but I realize thats unlikely in your case since it worked on your Ultra. I've tended to stay away from the Express model. I have a Roku Stick which I prefer and images look great on it.
